features/all-positions/AllLiquidations.tsx

```diff
@@ -211,116 +242,145 @@ const AllLiquidations = () => {
-            }}
-          >
-            <Arrow faded={page === maxPage ? true : false}>→</Arrow>
-          </div>
-        </PageButtons>
-      </TableContainer>
+                </Tooltip>
+
+                <TableCell align="right"></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {reformattedLiquidationData.map((id: string) => {
+                const liquidation = liquidations[id];
+                return (
+                  <StyledTableRow key={id}>
+                    <TableCell component="th" scope="row">
+                      <a
+                        href={getEtherscanUrl(liquidation.sponsor)}
+                        target="_blank"
+                      >
+                        {prettyAddress(liquidation.sponsor)}
+                      </a>
+                    </TableCell>
+                    <TableCell align="left">
+                      {prettyDate(Number(liquidation.liquidationTimestamp))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {translateLiquidationStatus(
+                        Number(liquidation.liquidationTimestamp),
+                        liquidation.status
+                      )}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(Number(liquidation.lockedCollateral))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(Number(liquidation.tokensLiquidated))}
+                    </TableCell>
+                    <TableCell align="right">
+                      {prettyBalance(
+                        getDisputablePrice(liquidation.maxDisputablePrice)
+                      )}
+                    </TableCell>
+
+                    <TableCell align="right">
+                      <MoreInfo onClick={() => handleOpenActionsDialog(id)}>
+                        ⋮
+                      </MoreInfo>
+                    </TableCell>
+                  </StyledTableRow>
+                );
+              })}
+            </TableBody>
+          </Table>
+          <PageButtons>
+            <div
+              onClick={() => {
+                setPage(page === 1 ? page : page - 1);
+              }}
+            >
+              <Arrow faded={page === 1 ? true : false}>←</Arrow>
+            </div>
+            {"Page " + page + " of " + maxPage}
+            <div
+              onClick={() => {
+                setPage(page === maxPage ? page : page + 1);
+              }}
+            >
+              <Arrow faded={page === maxPage ? true : false}>→</Arrow>
+            </div>
+          </PageButtons>
+        </TableContainer>
+        <LiquidationActionDialog
+          handleClose={() => setIsDialogShowing(!isDialogShowing)}
+          isDialogShowing={isDialogShowing}
+          liquidationId={liquidationId}
```

Comment:
nit: can we change `liquidationId` to `sponsorPlusId`

Reply:
sure & done